hooks/pre/10-reset_data.rb

```diff
@@ -5,10 +5,10 @@ def reset
   if foreman_installed?
 
     stop_services
-    start_services(['postgresql'])
+    start_services(['postgresql']) if local_postgresql?
     reset_database
```

Comment:
You should also check if Foreman is enabled.
```suggestion
    reset_foreman if foreman_server?
```

Note you also need to rename `reset_database`